db/schema.rb

```diff
@@ -202,10 +205,18 @@
   add_index "goodcity_requests", ["order_id"], name: "index_goodcity_requests_on_order_id", using: :btree
   add_index "goodcity_requests", ["package_type_id"], name: "index_goodcity_requests_on_package_type_id", using: :btree
 
+  create_table "goodcity_settings", force: :cascade do |t|
+    t.string "key"
+    t.string "value"
+    t.string "desc"
```

Comment:
What is "desc" ?

Reply:
description

Reply:
Might be confused with 'descending' in SQL!